### ISSUE TITLE
Forward Slash not possible in Title

### ISSUE THREAD

**Issue by Tuetenk0pp** [nextcloud/cookbook#368]
**Description**
It is impossible to use a forward slash `/` in the Title fiel. I have not tested other fields yet. 

**Reproduction**
1. Generate/edit a recipe.
2. use a `/` in the title field.
3. click confirm/apply
4. See error: the `/` is gone

**Expected behavior**
I would like to use the `/` in recipe titles

**Browser**
Firefox 82.0.2

**Comment by christianlupus** [comment]
Yes, that is due to the fact, that we are storing the recipe as a folder with the name of the recipe on the disk. The forward slash is a path separator causing trouble here. This name change is done [here](https://github.com/nextcloud/cookbook/blob/master/lib/Service/RecipeService.php#L1151) and [here](https://github.com/nextcloud/cookbook/blob/master/lib/Service/RecipeService.php#L135).

For now it is considered a feature, not a bug :roll_eyes:. Future changes might overcome this limitation but that is not a quick fix as it incorporates the rework of quite significant parts of the software.

Related issues: #340 #300 #299

**Comment by Tuetenk0pp** [comment]
I see. It's alright, I can use other symbols
